frontend/src/libs/hooks/use-toggle/use-toggle.hook.ts

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react';
+
+type ReturnType = [boolean, () => void];
```

Comment:
[ReturnType](https://www.typescriptlang.org/docs/handbook/utility-types.html#returntypetype) rename it